### ISSUE TITLE
Dam Boglars ToK unlocks

### ISSUE THREAD

**Issue by szejoza** [WarEmu/WarBugs#10566]
Finishing every step of Dam Boglars PQ doesn't update the last story part in the tome of knowledge
![sheykis_033](https://user-images.githubusercontent.com/18032193/31336736-783bc4a8-acf8-11e7-9c20-dd2a6cf71484.jpg)

**Comment by Lheana** [comment]
I have recently verified all ToK unlocks for PQ stages, to be sure the first stage triggers the ToK unlock for the first stage of the correct PQ for instance. But there are many errors in the ToK like missing description of the unlock so the line vanishes when the unlock is triggered, or 4 stages noted in ToK but only unlocks for 3 stages. All these errors can not be fixed yet, as we need client control.
I am aware of all these bugs, and I will fix it asap, but not possible not. Thus, no need to report these bugs.
Closing this report as not fixable yet.
